qmk_cli/subcommands/setup.py

```diff
@@ -35,7 +35,7 @@ def setup(cli):
     # Run `qmk_firmware/bin/qmk doctor` to check the rest of the environment out
     if qmk_firmware.exists():
         qmk_bin = qmk_firmware / 'bin' / 'qmk'
-        doctor = subprocess.run([sys.executable, str(qmk_bin), 'doctor'])
+        doctor = subprocess.run([sys.executable, str(qmk_bin), 'doctor', '-y' if cli.args.yes else ''])
```

Comment:
```suggestion
        doctor_cmd = [sys.executable, str(qmk_bin), 'doctor']
        if cli.args.yes:
            doctor_cmd.append('--yes')
        if cli.args.no:
            doctor_cmd.append('--no')
        doctor = subprocess.run(doctor_cmd)
```

Reply:
@skullydazed Shouldn't Line 41 be `elif` instead of `if`?

Reply:
No, if they pass both options to `qmk setup` we should probably pass both to `qmk doctor` as well.

Reply:
Why would you pass both `--yes` and `--no` at the same time? (Is this a situation of "read the ****ing code"?)

Reply:
It's more of a philosophical thing. I tend to avoid the use of `else if` when it's not necessary because that makes it easier to move code around and change it. I also tend to honor the user's wishes even if they don't make sense or seem contradictory, as long as I still have a clear and understandable result.

As a practical matter it doesn't make sense to pass both, and I believe (although I didn't check) that there's a check above to make sure the user doesn't do that.

Reply:
I did not handle the passing `--no` argument, because currently `doctor` does not have one.
I'm in favor of adding it and modifying this PR to pass it on.
I also don't have a problem with giving users the tools to shoot their foot off, but IMHO `--yes` and `--no` should be mutually exclusive.
We should either throw an error (preferred) or pass on only `--no`.